TOC.md

```diff
@@ -588,6 +588,7 @@
     - [Use TiDB to Read TiFlash Replicas](/tiflash/use-tidb-to-read-tiflash.md)
     - [Use TiSpark to Read TiFlash Replicas](/tiflash/use-tispark-to-read-tiflash.md)
     - [Use MPP Mode](/tiflash/use-tiflash-mpp-mode.md)
+    - [Disaggregated Storage and Compute Architecture and Amazon S3 Support](/tiflash/tiflash-disaggregated-and-s3.md)
```

Comment:
Better don't use Amazon S3 as title. S3 is more like a standard, not a specific product.